keyboards/cipulot/ec_pro2/readme.md

```diff
@@ -0,0 +1,27 @@
+# EC Pro2
+
+![EC Pro 2 PCB](https://i.imgur.com/uYOgTYo.png)
```

Comment:
This uses a smaller, but still high res image that is more suited for github's markdown rendering.
```suggestion
![EC Pro 2 PCB](https://i.imgur.com/uYOgTYoh.png)
```

Reply:
Done 👍